pages/linux/xman.md

```diff
@@ -0,0 +1,19 @@
+# xman
+
+> Manual page viewer for X Window System.
+
+- Start xman in three-button window:
+
+`xman`
+
+- Open helpfile other than default:
+
+`xman -helpfile {{filename}}`
+
+- Show both manual page and directory:
+
+`xman -bothshown`
+
+- Specify the geometry (size and location) of the top menu:
```

Comment:
Is this a commonly used option ? I would guess that the default geometry should be good enough for most cases.

Reply:
I guess not. Will remove